src/cs-gui/WebView.cpp

```diff
@@ -41,7 +41,7 @@ WebView::WebView(const std::string& url, int width, int height, bool allowLocalF
 
   int const targetFrameRate             = 60;
   browserSettings.windowless_frame_rate = targetFrameRate;
-  browserSettings.web_security          = allowLocalFileAccess ? STATE_DISABLED : STATE_ENABLED;
+  // browserSettings.web_security          = allowLocalFileAccess ? STATE_DISABLED : STATE_ENABLED;
```

Comment:
Why has this been disabled?

Reply:
Here the main discussion on bitbucket: https://bitbucket.org/chromiumembedded/cef/issues/3058/remove-cefbrowsersettingsweb_security

Reply:
From the link above this flag can only be set from command line. See [commit ](https://github.com/cosmoscout/cosmoscout-vr/pull/324/commits/33e455e01f6512fe8126af984842ff80063ff68b) for the fix